Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/example_commands.txt

```diff
@@ -0,0 +1 @@
+!ip ip="8.8.8.8"
```

Comment:
seems like template leftovers, please either supply relevant commands or delete the file